pyproject.toml

```
@@ -0,0 +1,142 @@
+[build-system]
+requires = ["setuptools>=61.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "wiki2beamer"
+version = "0.10.0"
+description = "Create LaTeX-beamer presentations with a wiki syntax"
+readme = "README.md"
+license = {text = "GPL-2.0-or-later"}
+requires-python = ">=3.7"
```

```suggestion
requires-python = ">=3.9"
```

as per:

https://devguide.python.org/versions/

---

Fixed

---

I thought about this some more, we may need to relax this a little if people on ancient pythons start complaining.

---

I'd say we should support non EOL versions of Python.  Otherwise it would be a bit challenging to maintain matrix CI test job as images for older versions would be hard to come by.  People who use older versions of Python can just grab previous version of the package.   If metadata is setup correctly this should happen automatically for them.